Tools/Scripts/webkitpy/w3c/wpt_runner_unittest.py

```diff
@@ -26,6 +26,7 @@
 from webkitpy.common.host_mock import MockHost
 from webkitpy.w3c.wpt_runner import WPTRunner, parse_args
 
+WPT_CHECKOUT_PATH = "/w3c-tests/web-platform-tests"
```

Comment:
Why not just `/wpt.git`? We should use the same name that cloning https://github.com/web-platform-tests/wpt gives you.